docs/ete-testing.md

```diff
@@ -0,0 +1,34 @@
+# End-to-end testing requirments
+
+StopSpot includes a variety of tests. Though most of them are unit tests that don't need to be configured, end-to-end test needs to be configured to be run succesfully. 
+
+**WARNING**: All the configurations must be prior to running pytest, and all of them are in `assets/config.json`. 
+
+## Overview
+
+*Note*: 
+  - `input information` - tables that contain raw information on which analyzers will be ran.
+  - `output information` - tables that contain analyzed information. 
+
+End-to-end test verifies correct output by the program. Thus, test file does the following (besides small checks):
+  `Step 1: ` - Read test data from csv and upload into test aperture (input information)
+  `Step 2: ` - Create test hive (output information)
+  `Step 3: ` - Run all data (input information) through analyzers and save data (output information) 
+  `Step 4: ` - Pull data from hive (output information)
+  `Step 5: ` - Validate that data from hive (output information) matches desired output
+  `Step 6: ` - Remove test aperture (containing test input information) and test hive (containing output information)
+
```

Comment:
For these to be displayed where each step is the start of a new line, you need to add two spaces to the end of the previous line.

Reply:
I think I fixed it